runtime/drive_handler.go

```diff
@@ -111,6 +111,9 @@ func (h *stubDriveHandler) createStubDrive(driveID, path string) error {
 			h.logger.WithError(err).Errorf("unexpected error during %v close", f.Name())
 		}
 	}()
+	if err := f.Chmod(0666); err != nil {
```

Comment:
Can we instead `Chown` this to the Jailer uid and then just leave the permissions at `0600` like they are set above? I don't think it would be good for any user on the box to be able to modify these stub drive files